README.md

```diff
@@ -68,25 +68,27 @@ These steps are to be completed by the author of the blog post.
 
    If you find there are a load of merge conflicts at this stage, see [Troubleshooting GitHub workflow](#troubleshooting-github-workflow).
 
-5. Request a build of the [draft openliberty.io site](https://draft-openlibertyio.mybluemix.net/blog/):
-    1. Sign in to [Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) with your GitHub account.
-    2. Click **More Options > Trigger Build**. Type `draft` in the **Branch** field, then click **Trigger custom build**.
+
+5. Currently, Travis is no longer building our non-prod sites. However, all the non-prod sites and more have been moved to IBM Cloud and now build automatically when the code is    merged into the respective branch.  The benefit is that we not only have the previous [draft site](https://draft-openlibertyio.mybluemix.net/blog/), but we now also have        [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) which only contains the blog content, allowing it to build and deploy much quicker.  However, the        drawback is that the builds are private and, therefore, their detailed build/deploy progress can't be tracked.  However, if you have access to the [Slack channel](https://app.slack.com/client/T15GKHBT4/C01GXGW1DGQ), then you can at least see when the builds are kicked started and when they finish.
```

Comment:
suggested edit:

Currently, Travis is no longer building our non-prod sites. All the non-prod sites and more have been moved to IBM Cloud and now build automatically whenever the a PR is  merged into their respective branch. These builds are private and, therefore, their detailed build/deploy progress can't be tracked.  However, if you have access to the [Slack channel](https://app.slack.com/client/T15GKHBT4/C01GXGW1DGQ), you can at least track when the builds start and finish. 

In addition to the main [draft site](https://draft-openlibertyio.mybluemix.net/blog/),  we now also have the [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/), which contains only the blog content, allowing it to build and deploy much quicker. However, since this site contains only the blog content, any links to other parts of openliberty.io will not resolve. If you need to review content that links to pages on openliberty.io that are not in the blogs, use the main [draft site](https://draft-openlibertyio.mybluemix.net/blog/).